src/components/FileUploader/FileUploader.js

```diff
@@ -96,6 +98,7 @@ export class FileUploaderButton extends Component {
           type="file"
           multiple={multiple}
           accept={accept}
+          name={name || ''}
```

Comment:
AFAIK React doesn’t set a property whose value is undefined. So I guess default empty string here is not needed?

Reply:
I guess the only reason why I left default as empty string is because I declared the propType to be of type String. Just a habit from using TypeScript and having it prevent the value from being undefined. I can remove the default value if you'd like

Reply:
For this case, might be worthwhile to have the empty string be set as the value in the `defaultProp` then 👍

Reply:
My latest commit removed these default values, but I will leave it to the team to decide whether or not you'd like to set a default name. It makes sense to me not to have a default name, and it doesn't seem like PropTypes will yell about it. I will defer to you guys on the final decision

Reply:
Since it's `name` for a form element, I think you're right that it shouldn't have a default value. For now, I'd just go with `name={name}` instead of giving it a default 👍 In the future we should make this prop type required, most likely.

Reply:
Sounds good, thanks for the input @asudoh and @joshblack. This branch is now ready for review